### ISSUE TITLE
Getting a compile error 

### ISSUE THREAD

**Issue by jeanbapt** [eshon/conference#5]
With Truffle 2.0.7, I am trying to compile MyConference.sol. I get this error. 
Any recent change on Solidity affecting this code ? 

Error compiling:

MyConference.sol:36:5: Warning: Return value of low-level calls not used.
                recipient.send(amount);
                ^--------------------^
Compiliation failed. See above.

**Comment by ephemeralriggs** [comment]
Yes. Recent security enhancements. Can you try changing the statement into this?:
if (!recipient.send(amount)) throw;

**Comment by robertock** [comment]
The suggestion worked.

**Comment by AdwindOne** [comment]
Error compiling:

Migrations.sol:7:3: Error: Expected token Semicolon got 'RBrace'
  }
  ^
Compilation failed. See above.
